spec.md

```diff
@@ -1399,6 +1500,127 @@ Errors](#service-broker-errors).
 }
 ```
 
+## Fetching a Service Instance
+
+If `"instance_retrievable" :true` is declared for a service in the
+[Catalog](#catalog-management) endpoint, brokers MUST support this endpoint for
+all plans of the service and this endpoint MUST be available immediately after
+the
+[Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances)
+endpoint returns `"state": "succeeded"` for a [Provisioning](#provisioning)
+operation.
+Otherwise, platforms SHOULD NOT attempt to call this endpoint under any
+circumstances.
+
+### Request
+
+##### Route
+`GET /v2/service_instances/:instance_id`
+
+`:instance_id` is the identifier of a previously provisioned instance.
+
+##### cURL
+```
+$ curl 'http://username:password@broker-url/v2/service_instances/:instance_id' -X GET -H "X-Broker-API-Version: 2.11"
+```
+
+### Response
+
+| Status Code | Description |
+| --- | --- |
+| 200 OK | The expected response body is below. |
+| 404 Not Found | MUST be returned if the service instance does not exist or if a provisioning operation is still in progress. The expected response body is `{}`. |
```

Comment:
While I prefer 404's, we don't use 404's in other places due to some concern about proxy errors - is that not a concern here?

Reply:
This is a comment for #159 I believe. Do we really want to not support 404s going forward because of possible proxy issues?

Reply:
I would love it if we could use 404's the way they were intended to be use :-)   but I'm just trying to be consistent with why we weren't allowed to use them in the past and why 410's were added.

Reply:
I prefer 404s regardless of proxy issue.